.eslintrc-browser.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": "./.eslintrc-base.json",
+
+  "env": {
+    // Note: don't set `"browser": true`; code in "src/" should be compatible with
+    // non-browser environments like Node.js with a custom window implementation
+    // like jsdom. All browser globals should be taken from window.
```

Comment:
No, it's not that bad. It's just an extra constraint that we have (and need to remember), which doesn't buy us much (we don't care about unit tests in non-browser environments).

But I am fine leaving it as is.